drivers/common/virtio/virtio_pci.c

```diff
@@ -0,0 +1,829 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2022 NVIDIA Corporation & Affiliates
+ */
+
+#include <stdint.h>
```

Comment:
if you are moving the code from X file to Y file.
Do not copy, you have to delete the code from some file that from where you are moving.